src/pages/MonthlyGood/MonthlyGoodLandingPage.vue

```diff
@@ -314,14 +321,47 @@ export default {
 // Experiment Styles - GROW-11
 .mg-hero.experiment {
 	::v-deep .overlay-content {
+		top: 2rem;
+		bottom: auto;
+		transform: none;
+		@include breakpoint(large) {
+			margin-left: 1rem;
+		}
+
+		@include breakpoint(medium only) {
+			top: 17rem;
+		}
+
+		@include breakpoint(small only) {
+			top: 17rem;
+		}
```

Comment:
I recommend defining these rules like this:
```
...
::v-deep .overlay-content {
    top: 17rem;
    bottom: auto;
    transform: none;
    @include breakpoint(large) {
        margin-left: 1rem;
        top: 2rem;
    }
}
...
```
You can also do a similar thing for the `.overlay-column` rules below.